content/documentation/pages/2-concepts/3-streams.md

```diff
@@ -6,11 +6,14 @@ description: 'Stream Processing Framework and Concepts'
 
 # Stream Processing
 
-Stream processing, is defined as the processing of infinite amount of data without interaction or interruption.
-Use cases for stream processing are real-time credit card fraud detection or predictive analytics.
-
-Stream processing in Data Flow is implemented architecturally as a collection of independent event-driven streaming applications that connect using messaging middleware, for example RabbitMQ or Apache Kafka.
-The collection of application is referred to as a streaming data pipelines.
+Stream processing is defined as the processing of an unbounded amount of data without interaction or interruption.
+Business cases for stream processing include:
+- Real-time credit card fraud detection or predictive analytics.
+- Continuous integration with machine learning platforms
```

Comment:
CI as a thing doesn't fit well in ML context, though. Maybe you wanted to highlight online model training workflows? Also, the predictive component is already called-out in the previous bullet.

Maybe we could rephrase to highlight "Enterprise Data Integration Patterns"

Reply:
I was attempting to highlight a business case that Google showcases for Pub/Sub which is that you can continually deliver new data to train and improve Machine learning models. 

I can remove it if we'd rather not push into that area.